protocol/0028-GOVE-governance.md

```diff
@@ -455,11 +455,20 @@ Notes:
+The batch proposal is a wrapper containing the same `reference` and `rationale` fields as a standard `ProposalSubmission` alongside a repeated list of `ProposalSubmission`s. 
+The individual submissions keep their own rationale entries. 
 
-Validation should be applied by the protocol when accepting such a transaction that all proposals within the batch are of the same category for the purposes of ensuring voting thresholds and minimum voting periods can be uniquely determined. Additionally, the closing time of each proposal's voting period must be identical to ensure that a single voting period can be run to determine the result of all. The enactment timestamp, however, should be customisable and can be different for each proposal within the batch.
+Validation should be applied by the protocol when accepting such a transaction to verify that all proposals within the batch meet their own minimum voting periods (if not transaction is rejected).
+Additionally, the closing time of each proposal's voting period must be identical to ensure that a single voting period can be run to determine the result of all. 
+The enactment timestamp, however, should be customisable and can be different for each proposal within the batch, as long as the minimum enactment time of each individual proposal within the batch is respected.
 
-Once submitted, a single voting period should be run in which participants may place a single vote to approve/disapprove of the entire batch. It should not be possible to vote for components in the batch separately. If the batch fails to pass the vote, the entire batch should be discarded as with any other proposal. If the batch passes, each of the component proposals should be enacted at their enactment timestamp exactly as if each had been proposed and passed individually. The enactment order of two proposals in the batch with the same enactment timestamp does not need to be defined and should be considered indeterminate from a user's point-of-view.
+Once submitted, a single voting period should be run in which participants may place a single vote to approve/disapprove of the entire batch. It *must* not be possible to vote for components in the batch separately.
+Once the closing timestamp is reached each individual proposal within the batch is evaluated against the votes received. 
+If all individual proposals would pass (given the votes received, based on their individual rules on participation, majority, LP-ELS voting etc.) then the entire batch passes. 
+If even one proposal within the batch would fail then the entire batch fails. 
+
+If the batch passes, each of the component proposals should be enacted at their enactment timestamp exactly as if each had been proposed and passed individually. The enactment order of two proposals in the batch with the same enactment timestamp does not need to be defined and should be considered indeterminate from a user's point-of-view.
```

Comment:
The execution of the proposal should not be indeterminate, and follow simple rules like: order by timestamps, and their order in the batch proposal. I can imagine that some proposals would submit 2 proposals, like e.g: increase the validator set, then increase themultisig signer. This need to have some guarantees. Now yes you could just set them to enact at different that to cover this too. Now anyway, there will be determinism of some sort if all proposal are enacted at the same date.

Reply:
If two or more proposals are enacting at the same time:
- they must enact in order that they were proposed
- if any of them are in the same batch they will have the same proposal timestamp and must enact in the order that they appear in the proposal

Reply:
Added.